cuda_ext.py

```diff
@@ -43,19 +47,19 @@ def find_msvc():
 exllama_ext = load(
     name = extension_name,
     sources = [
-        os.path.join(library_dir, "exllama_ext/exllama_ext.cpp"),
-        os.path.join(library_dir, "exllama_ext/cuda_buffers.cu"),
-        os.path.join(library_dir, "exllama_ext/cuda_func/q4_matrix.cu"),
-        os.path.join(library_dir, "exllama_ext/cuda_func/q4_matmul.cu"),
-        os.path.join(library_dir, "exllama_ext/cuda_func/column_remap.cu"),
-        os.path.join(library_dir, "exllama_ext/cuda_func/rms_norm.cu"),
-        os.path.join(library_dir, "exllama_ext/cuda_func/rope.cu"),
-        os.path.join(library_dir, "exllama_ext/cuda_func/half_matmul.cu"),
-        os.path.join(library_dir, "exllama_ext/cuda_func/q4_attn.cu"),
-        os.path.join(library_dir, "exllama_ext/cuda_func/q4_mlp.cu"),
-        os.path.join(library_dir, "exllama_ext/cpu_func/rep_penalty.cpp")
+        os.path.join(library_dir, os.path.join(library_prefix, extension_name, "exllama_ext.cpp")),
```

Comment:
Consider using `pathlib.Path` for these, letting you do:
```python
library_dir = Path("<...>")
extension_dir = library_dir / extension_name
[extension_dir / subpath for subpath in [
  "extllama_ext.cpp",
  "cuda_buffers.cu",
  "cuda_func/q4_matrix.cu",
  ...
]]
```
Reducing a lot of the excess line-noise wasted here.